worlds/tits_the_3rd/items.py

```diff
@@ -63,21 +76,38 @@ def get_item_id(item_name: ItemName):
 equipment_table: Dict[str, TitsThe3rdItemData] = {
     ItemName.royal_spikes: TitsThe3rdItemData(102, ItemClassification.useful),
     ItemName.black_bangle: TitsThe3rdItemData(356, ItemClassification.useful),
+    ItemName.black_bangle_plus: TitsThe3rdItemData(357, ItemClassification.useful),
     ItemName.glam_choker: TitsThe3rdItemData(358, ItemClassification.useful),
+    ItemName.glam_choker_plus: TitsThe3rdItemData(359, ItemClassification.useful),
+    ItemName.white_bracelet: TitsThe3rdItemData(360, ItemClassification.useful),
+    ItemName.proxy_puppet: TitsThe3rdItemData(325, ItemClassification.useful),
+    ItemName.long_barrel_2: TitsThe3rdItemData(332, ItemClassification.useful),
+    ItemName.crimson_eye: TitsThe3rdItemData(394, ItemClassification.useful),
+    ItemName.pearl_earings: TitsThe3rdItemData(362, ItemClassification.useful),
+    # Weapons
+    ItemName.akashic_heart: TitsThe3rdItemData(1184, ItemClassification.useful),
+    ItemName.stun_gb: TitsThe3rdItemData(1274, ItemClassification.useful),
```

Comment:
Probably for a later PR, but we probably want to put boss logic around weapons and equipment (e.g. a boss is out of logic until you have 4 usable weapons of X level, as you would in the base game)